content/docs/1.4.0/best-practices.md

```diff
@@ -94,7 +94,7 @@ If you're using `ext4` as the filesystem of the volume, we recommend adding a li
 
 ## Volume Maintenance
 
-Using Longhorn's built-in backup feature is highly recommended. You can save backups to an object store (such as S3 and Azure), an NFS server, or an SMB or CIFS server. Saving to an object store is preferable, if possible, because it generally has better performance and reliability.  There is also the advantage of not having to mount and unmount the target, which can complicate failover and upgrades.
+Using Longhorn's built-in backup feature is highly recommended. You can save backups to an object store such as S3, or an NFS server. Saving to an object store is preferable, if possible, because it generally has better performance and reliability.  There is also the advantage of not having to mount and unmount the target, which can complicate failover and upgrades.
```

Comment:
```suggestion
Using Longhorn's built-in backup feature is highly recommended. You can save backups to an object store such as S3 or to an NFS server. Saving to an object store is preferable because it generally offers better performance and reliability. Another advantage is that you do not need to mount and unmount the target, which can complicate failover and upgrades.
```

Reply:
Done.